src/Http/RequestFactory.php

```diff
@@ -33,6 +34,25 @@ class RequestFactory
 	/** @var array */
 	private $proxies = [];
 
+	/** @var callable[] of function (Request $request): mixed */
+	private $bodyParsers = [];
+
+
+	public function __construct()
+	{
+		$this->addBodyParser('application/x-www-form-urlencoded', function (Request $request) {
+			return $request->getPost();
```

Comment:
Files are not send with application/x-www-form-urlencoded mime type but with multipart/form-data.